<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Runs::FindRunnerWidget</class>
 <widget class="QWidget" name="Runs::FindRunnerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>656</width>
    <height>117</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="FindRunnerEdit" name="edFindRunner">
     <property name="placeholderText">
      <string>Name, registration, SI</string>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="cbxUseSIInNextStages">
     <property name="text">
      <string>Use this SI card also in next stages</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FindRunnerEdit</class>
   <extends>QLineEdit</extends>
   <header>src/findrunneredit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>edFindRunner</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
